lzy-api/src/main/proto/ai/lzy/v1/workflow/workflow-service.proto

```diff
@@ -127,3 +129,15 @@ message GetAvailablePoolsRequest {
 message GetAvailablePoolsResponse {
   repeated VmPoolSpec poolSpecs = 1;
 }
+
+
+// ==================== GetStorage =====================
+
+message GetStorageRequest {
+  // if not set, creds for internal SnapshotStorage are returned
+  /* optional */ string bucketName = 1;
```

Comment:
we can only obtain credits from internal storage only

I absolutely don't like the idea of making a credentials provider out of lzy

Reply:
summon @mrMakaronka

Reply:
I agree, we can only return some default storage config that includes bucket. Bucket definitely should not be a parameter here.